molgenis/molgenis-emx2
# feat(admin): add user role management

What are the main changes you did:
- closes https://github.com/molgenis/GCC/issues/208
- Adds the ability to change user roles on schema's 
- Adds a first (incomplete) version of a tailwind interface for admin tools

how to test:
- Run the ui app in dev mode
- Go to https://preview-emx2-pr-4248.dev.molgenis.org/apps/ui/
- Login as admin
- Click on the newly appeared admin button
- play around with role management by editing a user.


Other thing that work: 
- User creation/deletion 
- Updating user password, disabling users 
- Showing tokens
- 
Not implemented: token management

Other tests:
- go to localhost:3000/admin
- see that your redirected to the login screen

todo:
